### PR TITLE
Add note about removing /var/lib/sensu/sensu-backend/etcd in cluster guide

### LINKED ISSUE #2191: Elaborate on need to remove /var/lib/sensu/sensu-backend/etcd when clustering
We don't currently call out that /var/lib/sensu/sensu-backend/etcd needs to be removed if a member has been started prior to being clustered. We should add a note to the clustering guide that calls this out.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds NOTE callout about removing contents of `/var/lib/sensu/sensu-backend/etcd` for members previously started in "Run a Sensu cluster" guide.

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2191

## Review Instructions
- Is the text of the callout phrased correctly?
- Is the callout in the right place in the clustering guide?

### CODE REVIEW

File: content/sensu-go/5.18/guides/clustering.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ You can configure a Sensu cluster in a couple different ways &mdash; we'll show
 
 We also recommend using stable platforms to support your etcd instances (see [etcd's supported platforms][5]).
 
+_**IMPORTANT**: If any of your cluster members was started **before** being clustered, remove that member's path to state storage (`/var/lib/sensu/sensu-backend/etcd`) before continuing with configuration._
```
Comment (cwjohnston) [inline content/sensu-go/5.18/guides/clustering.md]:
```suggestion
{{% notice note %}}
**NOTE**: If a cluster member is started before it is configured to join a cluster, the member will persist its prior configuration to disk. For this reason, you must remove any previously started member's etcd data by stopping sensu-backend and deleting the contents of /var/lib/sensu/sensu-backend/etcd before proceeding with cluster configuration.
{{% /notice %}}
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Suggested some clarifications on the important note, but it may be too verbose now. WDYT?

File: content/sensu-go/5.18/guides/clustering.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.18/guides/clustering.md]: How does this sound? It is a bit shorter.

"If a cluster member is started **before** it is configured to join a cluster, the member will persist its prior configuration to disk. For this reason, you must remove any previously started member's etcd data by stopping sensu-backend and deleting the contents of `/var/lib/sensu/sensu-backend/etcd` before proceeding with cluster configuration."

Comment (hillaryfraley) [reply content/sensu-go/5.18/guides/clustering.md]: ...and if that note is good, should this be propagated back to 5.15?

Comment (cwjohnston) [reply content/sensu-go/5.18/guides/clustering.md]: @hillaryfraley I think your suggestion works, and I think this change is appropriate for all currently supported versions.

Comment (hillaryfraley) [reply content/sensu-go/5.18/guides/clustering.md]: Thank you!

**Pull request merged**